src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -82,6 +82,8 @@
 
     <h2>Translation Messages</h2>
 
+    {% block translation_messages %}
```

Comment:
Yeah. Inside we have blocks named "missing_messages", "fallback_messages", and "defined_messages". So naming "translation_messages" to "messages" makes sense.